README.md

```diff
@@ -1,26 +1,27 @@
 
-Going forward, new modules will need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules will no longer be accepted. To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
+> [!IMPORTANT]
+> Going forward, new modules need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules are no longer accepted. To propose a new AVM module, you can file a [AVM Module Proposal](https://aka.ms/AVM/ModuleProposal).
 
-Existing non-AVM modules will be retired or migrated to AVM. To provide continued access for existing customers, non-AVM modules formerly published in the registry will be kept there indefinitely, but their source code will be replaced with an informational notice and a pointer to their successor in AVM, when applicable. Over time, VS code intellisense support for the old, non-AVM modules will also be removed - while existing references will keep working.
-
-The AVM core team will orchestrate this transition, starting with developing a migration/retirement plan together with the owners of the modules impacted.
+Existing non-AVM modules have been retired or migrated to AVM. To provide continued access for existing customers, non-AVM modules formerly published in the registry will be kept there indefinitely, but their source code has been replaced with an informational notice and a pointer to their successor in AVM, when applicable. Visual Studio Code IntelliSense support for the old, non-AVM modules have also been removed - while existing references will keep working.
 
 For the list of available and planned AVM modules, please visit the [AVM Module Index](https://aka.ms/AVM/ModuleIndex) pages.
 
 # Bicep Registry Modules
 
-This repo contains the source code of all currently available Bicep modules in the Bicep public module registry.
+This repo contains the source code of all currently available Bicep modules in the Bicep public module registry (i.e., all AVM Bicep modules).
 
 ## Available Modules
 
-To view available modules and their versions, go to Bicep Registry Module Index ([https://aka.ms/br-module-index](https://aka.ms/br-module-index)).
+To view all available AVM Bicep modules, go to AVM Bicep Module Index ([https://aka.ms/AVM/ModuleIndex/Bicep](https://aka.ms/AVM/ModuleIndex/Bicep)).
 
 ## Contributing
 
-We only accept contributions from Microsoft employees at this time. Teams within Microsoft can refer to [Contributing to Bicep public registry](./CONTRIBUTING.md) for information on contributing modules. External customers can propose new modules or report bugs by opening an [issue](https://github.com/Azure/bicep-registry-modules/issues).
```

Comment:
What about previously linked pages, such as [./CONTRIBUTING.md](./CONTRIBUTING.md) should we retire it as well? 
I'd suggest at least to add a notice at the top mentioning the content is outdated and will be removed next.

Reply:
Here's the new PR, taking care of this: https://github.com/Azure/bicep-registry-modules/pull/1448